opennms-base-assembly/src/main/filtered/etc/alarmd/drools-rules.d/alarmd.drl

```diff
@@ -50,10 +50,13 @@ end
     <!-- Find all alarms that can be cleared -->
     <trigger name="selectAlarmsToClear" operator="&gt;=" row-count="1" >
       <statement>
-        SELECT *, now() AS _ts FROM alarms as problems
-        WHERE alarmType=1
-          AND severity &gt; 2
-          AND reductionKey IN (SELECT clearKey FROM alarms WHERE alarmType=2 AND lastEventTime &gt; problems.lastEventTime)
+        SELECT A1.*, now() AS _ts FROM alarms A1
```

Comment:
This is in a comment block - the previous vacuumd statements were only copied here as a reference.

Reply:
I was wondering too. Then neither the issue nor the changes make any sense to me, right? Should I close the PR & the issue?

Reply:
Yeah, it's no longer a problem now that we've moved to Drools.

Earlier releases could benefit from the change, but we don't want to modify the configuration files there. I'd say close it.